app/services/slack_notification_service.rb

```diff
@@ -5,14 +5,53 @@ class SlackNotificationService
                      'labeled' => :filter_on_hold_labeled_action,
                      'closed' => :filter_closed_action }.freeze
 
-  CHANNEL = '#code-review'.freeze
+  DEFAULT_CHANNEL = '#code-review'.freeze
+
+  LANGUAGES = {
```

Comment:
Not sure if we want to have so many channels, I would only with the ones that we consider right now as part of our stack and send the rest to the DEFAULT CHANNEL. 

Let say: 
{ Rails, Node, Python, React, Flutter, Kotlin, React Native, Swift } 

Also, what happens if the channel that Slack try to send the message, is not created? Is this creating the channel or it just raises an error?

Reply:
Ok, I'll reduce it so.
It has all possible languages supported by Github, so it always will find a channel, but I will change it for something better.